dom-lt.cabal

```diff
@@ -35,9 +36,9 @@ library
   ghc-options:      -O2 -funbox-strict-fields
   default-extensions: RankNTypes
   build-depends:
-      base >= 4.9 && < 5
+      base >= 4.3 && < 5
     , array
-    , containers >= 0.5 && < 0.7
+    , containers
```

Comment:
restore the upper bound and add 0.4.2.0 as the lower bound, which is the version that first defines NFData : https://hackage.haskell.org/package/containers-0.4.2.0